<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(65, 65, 65, 0.4);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="pos_image" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(202, 202, 202, 0.3);</string>
    </property>
    <widget class="QLabel" name="charimage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>image position = [CharacterCode]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pos_nickname" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QLabel" name="nickname">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>한동그라미</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="rankpoint">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  mmr = 7645</string>
    </property>
   </widget>
   <widget class="QLabel" name="ranknum">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  rank = 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="tierimage">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>tierimage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Userstat">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  User stat</string>
    </property>
   </widget>
   <widget class="QLabel" name="Gamestat">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[totalGames]전 [totalWins]승 승률</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgKill">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>average Kills</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgAssistant">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>average Assistants</string>
    </property>
   </widget>
   <widget class="QLabel" name="Charstat">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>281</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>  Character stat</string>
    </property>
   </widget>
   <widget class="QLabel" name="charusage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>플레이 횟수 : [usage]</string>
    </property>
   </widget>
   <widget class="QLabel" name="charIntop3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>in Top3 : [top3]</string>
    </property>
   </widget>
   <widget class="QLabel" name="charavgRank">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>average Rank : [averageRank]</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgRank">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>average Rank : [averageRank]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
